src/pages/home/report/ReportActionCompose.js

```diff
@@ -78,8 +78,11 @@ class ReportActionCompose extends React.Component {
             this.comment = this.props.comment;
         }
 
-        // When any modal goes from visible to hidden, bring focus to the compose field
-        if (prevProps.modal.isVisible && !this.props.modal.isVisible) {
+        // When any modal goes from visible to hidden or when the report ID changes, bring focus to the compose field
+        if (
```

Comment:
I don't think we separate lines for ifs something like this is closer to our guidelines I believe
``` javascript
if ((prevProps.modal.isVisible && !this.props.modal.isVisible)
    || (prevProps.reportID !== this.props.reportID) ) {
```

Would be otherwise happy :)

Reply:
never mind that, clearly I have not worked on the this repo for a while 😅

Reply:
I was wondering what y'all like to do in these situations! haha

I got this style from: https://github.com/airbnb/javascript#control-statements

Here, they reference this example:

```js
// good
if (
  (foo === 123 || bar === 'abc')
  && doesItLookGoodWhenItBecomesThatLong()
  && isThisReallyHappening()
) {
  thing1();
}
```

Anyway, let me know what you think! I'm happy to modify :D

Reply:
yeah, you are tottally right, sorry for that, I haven't open this repo in a while, this does seem like the way we do it in this project, keeping consitency is important, lets continue to do that

Reply:
Ok great, thanks for asking anyway!